Sources/Testing/ExitTests/SpawnProcess.swift

```diff
@@ -136,7 +136,7 @@ func spawnExecutable(
             // standardized in POSIX.1-2024 (see https://pubs.opengroup.org/onlinepubs/9799919799/functions/posix_spawn_file_actions_adddup2.html
             // and https://www.austingroupbugs.net/view.php?id=411).
             _ = posix_spawn_file_actions_adddup2(fileActions, fd, fd)
-#if canImport(Glibc)
```

Comment:
Add !OpenBSD while you're add it? SwiftTesting also has a bunch of existing OpenBSD conditionals.

Reply:
I've attempted to support OpenBSD but it's entirely untested. It's unclear to me if it would have a module named Glibc (its stdlib is not GNU.)